src/sentry/models/auditlogentry.py

```diff
@@ -94,8 +96,11 @@ def _apply_actor_label(self):
                 # TODO(hybridcloud) This requires an RPC service.
                 self.actor_label = self.actor_key.key
             else:
-                capture_exception(
-                    Exception("Expected there to be a user or actor key for audit logging")
+                logger.error(
+                    "Expected a user or actor key for audit log",
+                    extra={
```

Comment:
Should we throw in `stack_info=True` here? I'm not sure how sentry will render it since we don't use it anymore, but might be worth a shot?

Documented here https://docs.python.org/3/library/logging.html#logging.Logger.debug